internal/describer/object_test.go

```diff
@@ -110,7 +110,7 @@ func TestObjectDescriber(t *testing.T) {
 			)))
 
 	expected := component.ContentResponse{
-		Title: component.Title(component.NewLink("", "object", "."), component.NewText("pod")),
+		Title: component.Title(component.NewText("pod")),
```

Comment:
https://github.com/vmware-tanzu/octant/pull/1716/files#diff-f9af2246ccbdfd0724c02744f100700f43dd56d59254ac905a9a42d92b8ddba5L60 isn't meaningful at anytimes here

Reply:
I wanted to keep that test because that's still used to set title in some cases for example table headers.